src/test_driver/linux-cirque/test-mobile-device.py

```diff
@@ -95,7 +95,7 @@ def run_controller_test(self):
         for device_id in server_ids:
             self.logger.info("checking device log for {}".format(
                 self.get_device_pretty_id(device_id)))
-            self.assertTrue(self.sequenceMatch(self.get_device_log(device_id).decode('utf-8'), ["LightingManager::InitiateAction(ON_ACTION)", "LightingManager::InitiateAction(OFF_ACTION)"]),
+            self.assertTrue(self.sequenceMatch(self.get_device_log(device_id).decode('utf-8'), ["LightingManager::InitiateAction(ON_ACTION)", "LightingManager::InitiateAction(OFF_ACTION)", "Cannot found cluster 0x6 on endpoint 0xe9"]),
```

Comment:
Would this test not fail as the log was now updated ?

Reply:
Yes, I forgot to update this piece, thanks!